src/Compilers/CSharp/Portable/CSharpCodeAnalysis.csproj

```diff
@@ -892,17 +893,16 @@
     </BoundTreeDefinition>
     <Compile Include="$(IntermediateOutputPath)BoundNodes.xml.Generated.cs" />
     <Compile Include="$(IntermediateOutputPath)ErrorFacts.Generated.cs" />
-    <Compile Include="$(IntermediateOutputPath)Syntax.xml.Generated.cs" />
+    <Compile Include="$(IntermediateOutputPath)Syntax.xml.Syntax.Generated.cs" />
```

Comment:
@jasonmalinowski 

Actually the order of definition and consumption is backwards here.  This variables are defined as a part of the `Imports.targets` chain which occurs at the bottom of the file.  Hence it's not available here. 

I considered adding the `Compile` items in `GenerateComplireInternals.targets` but I could only envision that working as a conditional `ItemGroup`.  In the past you've advised against those, at least for references. 

Happy to make this better but don't see a good path for it.